load.go

```diff
@@ -365,14 +371,17 @@ func (s *Spec) FillDefaults() {
 
 func (s Spec) Validate() error {
 	for name, src := range s.Sources {
+		if strings.ContainsRune(name, os.PathSeparator) {
```

Comment:
On this note, have we specifically defined what are and are not valid source names?

Reply:
We have not, but should indeed.